bands/templates/bands/w9.html

```diff
@@ -10,6 +10,7 @@
 
 <form method="post" action="w9" enctype="multipart/form-data">
 <input type="hidden" name="id" value="{{ band.id }}" />
```

Comment:
One general note about all of these forms: we're going to want people to be able to edit their existing data after the form was filled out for the first time.  At present that won't work because the form doesn't load the existing data.

My suggestion is that instead of

```
<input type="hidden" name="id" value="{{ band.id }}" />
```

we instead say something like

```
<input type="hidden" name="band_id" value="{{ band.id }}" />
<input type="hidden" name="id" value="{{ bio.db_id }}" />
```

That should do what we want, since a model object's ``.db_id`` property returns ``None`` if the object hasn't yet been saved and returns the id if the object came from the database.